Classes/Creator/Command/CommandCreator.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the package stefanfroemken/ext-kickstarter.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace StefanFroemken\ExtKickstarter\Creator\Command;
+
+use PhpParser\BuilderFactory;
+use PhpParser\Node\Stmt\Return_;
+use StefanFroemken\ExtKickstarter\Information\CommandInformation;
+use StefanFroemken\ExtKickstarter\PhpParser\NodeFactory;
+use StefanFroemken\ExtKickstarter\PhpParser\Structure\ClassStructure;
+use StefanFroemken\ExtKickstarter\PhpParser\Structure\DeclareStructure;
+use StefanFroemken\ExtKickstarter\PhpParser\Structure\FileStructure;
+use StefanFroemken\ExtKickstarter\PhpParser\Structure\MethodStructure;
+use StefanFroemken\ExtKickstarter\PhpParser\Structure\NamespaceStructure;
+use StefanFroemken\ExtKickstarter\PhpParser\Structure\UseStructure;
+use StefanFroemken\ExtKickstarter\Traits\FileStructureBuilderTrait;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+
+class CommandCreator implements CommandCreatorInterface
+{
+    use FileStructureBuilderTrait;
+
+    private NodeFactory $nodeFactory;
+
+    private BuilderFactory $builderFactory;
+
+    public function __construct(NodeFactory $nodeFactory)
+    {
+        $this->nodeFactory = $nodeFactory;
+        $this->builderFactory = new BuilderFactory();
+    }
+
+    public function create(CommandInformation $commandInformation): void
+    {
+        GeneralUtility::mkdir_deep($commandInformation->getCommandPath());
+
+        $commandFilePath = $commandInformation->getCommandFilePath();
+        $fileStructure = $this->buildFileStructure($commandFilePath);
+
+        if (!is_file($commandFilePath)) {
+            $this->addClassNodes($fileStructure, $commandInformation);
+            file_put_contents($commandFilePath, $fileStructure->getFileContents());
+        }
+    }
+
+    private function addClassNodes(FileStructure $fileStructure, CommandInformation $commandInformation): void
+    {
+        $fileStructure->addDeclareStructure(
+            new DeclareStructure($this->nodeFactory->createDeclareStrictTypes())
+        );
+        $fileStructure->addUseStructure(
+            new UseStructure($this->nodeFactory->createUseImport('Symfony\Component\Console\Attribute\AsCommand'))
+        );
+        $fileStructure->addUseStructure(
+            new UseStructure($this->nodeFactory->createUseImport('Symfony\Component\Console\Command\Command'))
```

Comment:
why are you using strings for the classes instead of something like Command::class?

Reply:
Don't remember anymore. IMO, because `::class` results in `\Vendor\Ext\WhatEver` instead of `Vendor\Ext\WhatEver`